hugofs/decorators.go

```diff
@@ -191,6 +191,26 @@ func (l *baseFileDecoratorFile) Readdir(c int) (ofi []os.FileInfo, err error) {
 	for _, dirname := range dirnames {
 		filename := dirname
 
+		ext := filepath.Ext(filename)
+		baseName := filepath.Base(filename)
+		istemp := strings.HasSuffix(ext, "~") ||
```

Comment:
would it be possible to make this data driven / config driven? I feel like instituting a pattern like `.hugoignore` supporting regular expressions might make this pattern more extensible, rather than inlining support for a bunch of editors of the implementer's choice.